tardis/adapters/sites/moab.py

```diff
@@ -72,12 +72,29 @@ def __init__(self, machine_type: str, site_name: str):
             remote_resource_uuid="JobID", resource_status="State"
         )
 
+        # see job state codes at https://computing.llnl.gov/tutorials/moab/#JobStates
         translator_functions = StaticMapping(
             State=lambda x, translator=StaticMapping(
+                BatchHold=ResourceStatus.Stopped,
+                Canceling=ResourceStatus.Running,
+                Completed=ResourceStatus.Deleted,
+                Deffered=ResourceStatus.Booting,
+                Depend=ResourceStatus.Error,
+                Dependency=ResourceStatus.Error,
                 Idle=ResourceStatus.Booting,
+                JobHeldUser=ResourceStatus.Stopped,
+                Migrated=ResourceStatus.Booting,
+                NotQueued=ResourceStatus.Error,
+                Priority=ResourceStatus.Booting,
+                Removed=ResourceStatus.Deleted,
+                Resources=ResourceStatus.Booting,
                 Running=ResourceStatus.Running,
-                Completed=ResourceStatus.Deleted,
-                Canceling=ResourceStatus.Running,
+                Staging=ResourceStatus.Booting,
+                Starting=ResourceStatus.Booting,
+                Suspended=ResourceStatus.Stopped,
+                SystemHold=ResourceStatus.Stopped,
+                TimeLimit=ResourceStatus.Stopped,
```

Comment:
We have to cautious with the state `ResourceStatus.Stopped`, since the system will still try to remove the job by executing `canceljob`. Are we sure that a jobs that run into the `TimeLimit` or be `Vacated` can be canceled? Otherwise I would propose to assign it to state `ResourceStatus.Deleted`. This should be no problem for held or suspended jobs, though.

Reply:
As @stefan-k reported on gitter mapping `TimeLimit` and `Vacated` to `ResourceStatus.Deleted` in the MOAB site adapter seems to solve their problems.

Reply:
I've changed that. Thanks for the input, @stefan-k!